### PR TITLE
Implement Instructions for Manual Testing for ecreate, eaddv, elistv and eremovev

### PR DESCRIPTION
**Pull request by jordankanghm**
as above

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -765,11 +765,95 @@ testers are expected to do more *exploratory* testing.
...
+   1. Prerequisites: The event list panel must contain at least one event.
+   
+   1. Test case: `elistv 1`<br>
+      Expected: Volunteer list panel shows all volunteers currently assigned to the event. Status message shows a success message as well as the number of volunteers assigned to the event.
+   
+   1. Test case: `elistv`<br>
+      Expected: No change in the volunteer list panel. Error message for invalid command format is shown in the status message.
+      The outcome is the same for input commands with no index specified, or non-positive indexes.
+   
+   1. Test case: `elistv 3`<br>
```
Comment (tiongMax) [inline docs/DeveloperGuide.md:826]: Why is it 1 all the way tho?

Comment (brein62) [reply docs/DeveloperGuide.md:826]: that's fine, markdown processes it just fine

**Review by zacwong2151 — APPROVED** [review]
what does the fox say? 🦊

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -765,11 +765,95 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `ecreate`<br>
+      Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+      The outcome is the same when any of the compulsory parameters are missing from the input command.
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/`<br>
+      Expected: No change to the event list panel. Error message for the invalid field is shown in the status message<br>
+      The outcome is the same when all compulsory parameters are present and any argument is empty or invalid.
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/Packing food for the needy`, followed by
+                 `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/Packing food for the needy`<br>
```
[the remark this answers is not in the document]
Comment (ruo-x) [reply docs/DeveloperGuide.md:782]: nice one zac

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -765,11 +765,95 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: No change to the event list panel. Error message for duplicate events is shown in the status message<br>
+      The outcome is the same when all compulsory parameters are present, all arguments are valid, and an event with the same name already exists in the event list.
+   
+### Adding a volunteer to an event
+1. Adding a volunteer which is being shown to an event which is being shown
+
+   1. Prerequisites: The event list must contain at least one event and the volunteer list must contain at least one volunteer.
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:789]: i think this can be event list panel and volunteer list panel instead

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -765,11 +765,95 @@ testers are expected to do more *exploratory* testing.
...
+   
+   1. Test case: `elistv 1`<br>
+      Expected: Volunteer list panel shows all volunteers currently assigned to the event. Status message shows a success message as well as the number of volunteers assigned to the event.
+   
+   1. Test case: `elistv`<br>
+      Expected: No change in the volunteer list panel. Error message for invalid command format is shown in the status message.
+      The outcome is the same for input commands with no index specified, or non-positive indexes.
+   
+   1. Test case: `elistv 3`<br>
+      Additional Prerequisites: The event list panel currently has only 2 events.<br>
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:827]: as for this u can consider copying the original format in the DG its a lot clearer imo:
`Test case: `eshow x` (where x is larger than the list size)`

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -765,11 +765,95 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: No change in the volunteer list panel. Error message for invalid event index is shown in the status message.<br>
+      The outcome is the same for input commands where the index specified is larger than the number of events in the event list panel.
+
+### Removing a volunteer from an event
+1. Removing a volunteer which is being shown from an event which is being shown
+
+    1. Prerequisites: The event list must contain at least one event and the volunteer list must contain at least one volunteer.
+
+    1. Test case: `eremovev eid/1 vid/1`<br>
+       Prerequisites:
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:837]: why some its `additional prerequisites` but some is just `prerequisites`?

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -765,11 +765,95 @@ testers are expected to do more *exploratory* testing.
...
+           
+       Expected: The volunteer is removed from the event. Details of the updated event and its current number of assigned volunteers are shown in the status message.
+
+    1. Test case: `eremovev eid/ vid/`<br>
+       Expected: No change to the event list or volunteer list panels. Invalid command format error is shown in the status message.<br>
+       The outcome is the same when any of the parameters are missing from the input command, or when their arguments are empty or invalid.
+
+    1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy` and
+                  `vcreate n/tom p/12345678 e/tom@gmail.com`, followed by `eremovev eid/1 vid/1`<br>
+       Expected: No change to the event list or volunteer list panels. Error message for volunteer not currently assigned to event is shown in the status message.<br>
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:849]: felt like `Error message for volunteer not currently assigned to event is shown in the status message.`  was a little confusing, maybe can just say `error message for invalid volunteer is shown in the status message`

**Review by ruo-x — CHANGES_REQUESTED** [review]
bad job.

File: docs/DeveloperGuide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tiongMax) [reply docs/DeveloperGuide.md:782]: Fuish zac

[the remark this answers is not in the document]
Comment (zacwong2151) [reply docs/DeveloperGuide.md:782]: good one zac

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:826]: we are number one

File: docs/DeveloperGuide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jordankanghm) [reply docs/DeveloperGuide.md:782]: HAHAHAHAH

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:789]: true

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:837]: the only other place i used the additional prerequisites is in the eaddv positive test case. cos need to ensure there's no duplicate volunteers or clashing events

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:849]: mayb u can try expanding ur brain?

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -1097,6 +1097,88 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `eaddv eid/1 vid/1`, followed by `eaddv eid/1 vid/1`<br>
+      Expected: No change to the event list or volunteer list panels. Error message for duplicate volunteer is shown in the status message<br>
+      The outcome is the same when any volunteer is added to an event they are already assigned to.
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy` and
+                 `ecreate n/Clean park r/20 cleaner sd/23/10/2023 2000 ed/23/10/2023 2300 l/serangoon dsc/Clean the local park`, followed by
+                 `eaddv eid/1 vid/1` and `eaddv eid/2 vid/1`<br>
+      Note that the two events created are clashing with each other.<br>
+      Expected: No change to the event list or volunteer list panels. Error message for clashing events is shown in the status message<br>
+      The outcome is the same when any volunteer is added to an event which clashes with at least one of their pre-existing events.
```
Comment (brein62) [inline docs/DeveloperGuide.md:1144]: looks lit but i think need to have a prerequisite that event list is empty? else they won't know what `eid/1` and `eid/2` refer to. or else you can do something like `eid/X` like what i did idk hahah

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -1097,6 +1097,88 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: The volunteer is removed from the event. Details of the updated event and its current number of assigned volunteers are shown in the status message.
+
+    1. Test case: `eremovev eid/ vid/`<br>
+       Expected: No change to the event list or volunteer list panels. Invalid command format error is shown in the status message.<br>
+       The outcome is the same when any of the parameters are missing from the input command, or when their arguments are empty or invalid.
+
+    1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy` and
+                  `vcreate n/tom p/12345678 e/tom@gmail.com`, followed by `eremovev eid/1 vid/1`<br>
+       Expected: No change to the event list or volunteer list panels. Error message for invalid volunteer is shown in the status message.<br>
+       The outcome is the same when any volunteer is removed from an event they are not currently assigned to.
```
Comment (brein62) [inline docs/DeveloperGuide.md:1180]: same issue as above!

**Review by brein62 — CHANGES_REQUESTED** [review]
just a small issue so that the testers won't be confused!

File: docs/DeveloperGuide.md (hunk 7 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:1144]: aight

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (ruo-x) [reply docs/DeveloperGuide.md]: oi pg13 language oni hor

**Review by ruo-x — APPROVED** [review]
mkay good job

**Pull request merged**